dbtmetabase/__init__.py

```diff
@@ -69,9 +85,6 @@ def export(
         assert (
             schema and not schemas_excludes
         ), "Must target a single schema if using yaml parser, multiple schemas not supported."
-    assert bool(schema) != bool(
```

Comment:
This is actually fixing a bug that is available on existing `master` docs sync, right?

Reply:
I actually had a bit of confusion there and Im not fully settled on this particular change. But I'll offer what I know and curious as to your comments.

Schema is a required arg on `export_models` function as well as the `build_metadata_lookups`. In that function, the `schema` arg restricts documentation/foriegn_key/semantic_type propagation outwards to only Metabase tables that matched the passed schema. Author likely intended separate invocations per schema? This is independent of the dbt folder / manifest reader and I believe this was in place prior to our work. So if you ran it on your fork without passing schema (took another look at the fork to be sure), a string definitely needed to be passed to `export` and to `build_metadata_lookups`. 

So my question is it seems we have sort of two different intentions running side by side here. If someone can clarify the best case desired behavior and can correct (in the metabase client would be the place most likely).